administrator/components/com_associations/language/en-GB/en-GB.com_associations.ini

```diff
@@ -0,0 +1,11 @@
+; Joomla! Project
+; Copyright (C) 2005 - 2016 Open Source Matters. All rights reserved.
+; License GNU General Public License version 2 or later; see LICENSE.txt, see LICENSE.php
+; Note : All ini files need to be saved as UTF-8
+
+COM_ASSOCIATIONS="Multi-lingual Associations"
+COM_ASSOCIATIONS_XML_DESCRIPTION="Improved multi-lingual content management component"
+COM_ASSOCIATIONS_TITLE = "Associations"
+COM_ASSOCIATIONS_REFERENCE_ITEM = "Reference"
+COM_ASSOCIATIONS_ASSOCIATED_ITEM = "Target"
+COM_ASSOCIATIONS_HEADING_ASSOCIATION = "Association"
```

Comment:
you don't need to do this until midterm.

But the language variables are always alpha ordered.